### PR TITLE
Markdown bolded rendering fails

### PR DESCRIPTION
**Pull request by isolcat**
### Description
Markdown bold rendering failure, Chinese documents and English documents compared to see that the process of writing more than one space added

### Motivation
Fix Chinese documentation

### Additional details
THE PROBLEMS happend at the doc:https://developer.mozilla.org/zh-CN/docs/Web/CSS/transition

THE CORRECT reference at the doc:https://developer.mozilla.org/en-US/docs/Web/CSS/transition

### CODE REVIEW

File: files/zh-cn/web/css/transition/index.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ slug: Web/CSS/transition
 
 {{CSSRef}}
 
-**`transition` **[CSS](/zh-CN/CSS) 属性是 {{ cssxref("transition-property") }}，{{ cssxref("transition-duration") }}，{{ cssxref("transition-timing-function") }} 和 {{ cssxref("transition-delay") }} 的一个[简写属性](/zh-CN/docs/CSS/Shorthand_properties)。
+**`transition`**[CSS](/zh-CN/CSS) 属性是 {{ cssxref("transition-property") }}，{{ cssxref("transition-duration") }}，{{ cssxref("transition-timing-function") }} 和 {{ cssxref("transition-delay") }} 的一个[简写属性](/zh-CN/docs/CSS/Shorthand_properties)。
```
Comment (yin1999) [inline files/zh-cn/web/css/transition/index.md:8]:
```suggestion
**`transition`** [CSS](/zh-CN/CSS) 属性是 {{ cssxref("transition-property") }}、{{ cssxref("transition-duration") }}、{{ cssxref("transition-timing-function") }} 和 {{ cssxref("transition-delay") }} 的一个[简写属性](/zh-CN/docs/CSS/Shorthand_properties)。
```

**Review by yin1999 — CHANGES_REQUESTED** [review]

File: files/zh-cn/web/css/transition/index.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ slug: Web/CSS/transition
 
 {{CSSRef}}
 
-**`transition` **[CSS](/zh-CN/CSS) 属性是 {{ cssxref("transition-property") }}，{{ cssxref("transition-duration") }}，{{ cssxref("transition-timing-function") }} 和 {{ cssxref("transition-delay") }} 的一个[简写属性](/zh-CN/docs/CSS/Shorthand_properties)。
+**`transition`** [CSS](/zh-CN/CSS) 属性是 {{ cssxref("transition-property") }}、{{ cssxref("transition-duration") }}、{{ cssxref("transition-timing-function") }} 和 {{ cssxref("transition-delay") }} 的一个[简写属性](/zh-CN/docs/CSS/Shorthand_properties)。
```
Comment (yin1999) [inline files/zh-cn/web/css/transition/index.md:8]:
```suggestion
**`transition`** [CSS](/zh-CN/docs/Web/CSS) 属性是 {{ cssxref("transition-property") }}、{{ cssxref("transition-duration") }}、{{ cssxref("transition-timing-function") }} 和 {{ cssxref("transition-delay") }} 的一个[简写属性](/zh-CN/docs/CSS/Shorthand_properties)。
```

**Review by yin1999 — APPROVED** [review]
Thanks

**Comment by isolcat** [comment]
> 谢谢

没事 很高兴能为中文文档的更加完善填一份力 感谢你的合并

**Pull request merged**